src/main/scala/io/github/davidgregory084/ScalacOptions.scala

```diff
@@ -148,6 +152,14 @@ private[davidgregory084] trait ScalacOptions {
   def advancedOption(name: String, isSupported: ScalaVersion => Boolean = _ => true): ScalacOption =
     ScalacOption(s"-X$name", isSupported)
 
+  /** Advanced options (-X)
+    */
+  def advancedOption(
+    name: String,
+    arguments: List[String],
+    isSupported: ScalaVersion => Boolean = _ => true
```

Comment:
```suggestion
    isSupported: ScalaVersion => Boolean
```
I think you will need to remove the default argument here to fix:

> >  [error] /home/runner/work/sbt-tpolecat/sbt-tpolecat/src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala:30:12: in object ScalacOptions, multiple overloaded alternatives of method advancedOption define default arguments.
[error] The members with defaults are defined in trait ScalacOptions in package davidgregory084 and trait ScalacOptions in package davidgregory084.
[error]     object ScalacOptions extends ScalacOptions
[error]            ^